docs/maven-tutorial.md

```diff
@@ -0,0 +1,218 @@
+# OpenShift + Maven Tutorial
+
+While developing your application with Java, you'll most likely be building
+many, many, many times.  In order to shorten the build times of your pods,
+dependencies can be cached in local Maven repositories.  This tutorial will step
```

Comment:
@bparees Sorry I just noticed this comment.  Could you explain a bit more?  The .m2 dir goes into the image running the jee-sample?